moveit_ros/planning_interface/moveit_cpp/src/moveit_cpp.cpp

```diff
@@ -36,40 +36,35 @@
 
 #include <stdexcept>
 #include <moveit/moveit_cpp/moveit_cpp.h>
-#include <moveit/planning_scene_monitor/current_state_monitor.h>
-#include <moveit/common_planning_interface_objects/common_objects.h>
-
-#include <std_msgs/String.h>
 #include <tf2/utils.h>
 #include <tf2_ros/transform_listener.h>
-#include <ros/console.h>
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 
 namespace moveit
 {
 namespace planning_interface
 {
-constexpr char LOGNAME[] = "moveit_cpp";
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.ros_planning_interface.moveit_cpp");
```

Comment:
nit : it may be nice to make the logger a MoveItCpp class variable so that you can namespace it below the name of the Node that gets passed in.

Reply:
Worth a discussion, but for the release this won't happen as this follows guidelines we decided upon for the migration.